src/charm.py

```diff
@@ -119,13 +235,29 @@ def _peers(self) -> Relation:
              A :class:`ops.model.Relation` object representing
              the peer relation.
         """
-        return self.model.get_relation("postgresql-replicas")
+        return self.model.get_relation(PEER)
 
     def _get_postgres_password(self) -> str:
         """Get postgres user password."""
         data = self._peers.data[self.app]
         return data.get("postgres-password", None)
 
+    def _get_replication_password(self) -> str:
+        """Get replication user password."""
+        data = self._peers.data[self.app]
+        return data.get("replication-password", None)
+
+    def _unit_name_to_pod_name(self, unit_name: str) -> str:
+        """Converts unit name to pod name.
+
+        Args:
+            unit_name: name in "postgresql-k8s/0" format.
+
+        Returns:
+            pod name in "psotgresql-k8s-0" format.
```

Comment:
@marceloneppel , minor typo here. s/psotgresql/postgresql/

Reply:
Thanks for catching it @paulomach! I'll fix it.